src/app/[locale]/campagne-partenaire/[pollSlug]/_components/PartnerCampaignContent.tsx

```diff
@@ -0,0 +1,78 @@
+'use client'
+
+import type { PartnerCampaignType } from '@/adapters/cmsClient'
+import ButtonLink from '@/design-system/buttons/ButtonLink'
+import Hero from '@/design-system/layout/landingPage/Hero'
+import { useFetchPublicPoll } from '@/hooks/organisations/polls/useFetchPublicPoll'
+import Image from 'next/image'
+import type { ReactNode } from 'react'
+import PartnerCampaignHeader from './PartnerCampaignHeader'
+
+export default function PartnerCampaignContent({
+  pollSlug,
+  partnerCampaign,
+  partnersComponent,
+  faqComponent,
+}: {
+  pollSlug: string
+  partnerCampaign: PartnerCampaignType
+  partnersComponent: ReactNode
+  faqComponent?: ReactNode
+}) {
+  const { data: pollInfo } = useFetchPublicPoll({
+    pollIdOrSlug: pollSlug,
+  })
+
+  return (
+    <>
+      <PartnerCampaignHeader logoSrc={partnerCampaign.logo.url} />
+      <Hero
+        title={partnerCampaign.title}
+        className="pt-8 pb-0 md:pt-16 md:pb-20"
+        description={
+          <>
+            <div
+              dangerouslySetInnerHTML={{ __html: partnerCampaign.htmlContent }}
+            />
+
+            <ButtonLink
```

Comment:
Should not be displayed if no pollInfo... Request could be long and even error you know